### PR TITLE
Homework for DB-Week-1

### PR DESCRIPTION
**Pull request by Nusrath-Jahan**
1. Find out how many tasks are in the task table
2. Find out how many tasks in the task table do not have a valid due date
3. Find all the tasks that are marked as done
4. Find all the tasks that are not marked as done
5. Get all the tasks, sorted with the most recently created first
6. Get the single most recently created task
7. Get the title and due date of all tasks where the title or description contains database
8. Get the title and status (as text) of all tasks
9. Get the name of each status, along with a count of how many tasks have that status
10. Get the names of all statuses, sorted by the status with most tasks first

### CODE REVIEW

File: DataBase/DB-Week-1/week-1.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+--8.Get the title and status (as text) of all tasks
+SELECT title, status.name AS status FROM task JOIN status ON task.status_id = status.id;
+
+--9.Get the name of each status, along with a count of how many tasks have that status
+SELECT s.name, COUNT(t.status_id) as Total_Task
+FROM task t JOIN status s
+on t.status_id = s.id
+GROUP BY t.status_id;
+
+--10.Get the names of all statuses, sorted by the status with most tasks first
```
Comment (jpganz) [inline DataBase/DB-Week-1/week-1.sql:47]: this query is wrong, please take a second look at it

**Review by jpganz — CHANGES_REQUESTED** [review]

File: DataBase/DB-Week-1/week-1.sql (hunk 1 above)
Comment (Nusrath-Jahan) [reply DataBase/DB-Week-1/week-1.sql:49]: I appreciate your review. Could you look over my queries? I edited the line you pointed out a little bit. Is that right now?
Thank you

Comment (Nusrath-Jahan) [reply DataBase/DB-Week-1/week-1.sql:49]: Hello Mentor, Should I merge this branch?

Comment (jpganz) [reply DataBase/DB-Week-1/week-1.sql:49]: Hi @Nusrath-Jahan  I believe the query number 10 still needs some work

Comment (Nusrath-Jahan) [reply DataBase/DB-Week-1/week-1.sql:49]: Thank you for your reply!!
Could you kindly confirm that everything is okay now?

Comment (Nusrath-Jahan) [reply DataBase/DB-Week-1/week-1.sql:49]: Hello jpganz, 
Could you please check the number 10 task ?

**Pull request merged**